### PR TITLE
Simplify TextSpan creation

### PR DESCRIPTION
**Pull request by andy-ms**
Adds a helper function and uses existing helpers more.

### CODE REVIEW

File: src/services/textChanges.ts [hunk 1]
```diff
@@ -632,7 +632,7 @@ namespace ts.textChanges {
         }
 
         private computeSpan(change: Change, _sourceFile: SourceFile): TextSpan {
-            return createTextSpanFromBounds(change.range.pos, change.range.end);
+            return createTextSpanFromRange(change.range);
```
Comment (armanio123) [inline src/services/textChanges.ts]: Is this name the intended one? On other files the method is creatTextSpanFromStartEnd.

Comment (andy-ms) [reply src/services/textChanges.ts]: This and `createTextSpanFromStartEnd` are basically the same functions, only some data structures use `{ start, end }` and others use `{ pos, end }` for some reason.

Comment (armanio123) [reply src/services/textChanges.ts]: If both methods (FromStartEnd and FromRange) achieves the same goal, is it preferable to instead refactor to use the FromRange and not introduce a new method?

Comment (andy-ms) [reply src/services/textChanges.ts]: :+1: Changed this so that they call it `pos` consistently

**Pull request merged**